src/components/inputcomponents/ImageInput/SingleInput.js

```diff
@@ -1,5 +1,8 @@
 import React, { PropTypes } from 'react';
 import Dropzone from 'react-dropzone';
+import DropboxChooser from '../../imports/DropboxChooser';
+
+const APP_KEY = 'uajlwnyea4n8yme';
```

Comment:
Move this setting to [outCalls/config.js](https://github.com/Cloud-CV/cvfy-frontend/blob/master/outCalls/config.js) and add the documentation for it. The person who deploys this app should provide this, not us.